packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -209,7 +209,7 @@ export default class Meetings extends WebexPlugin {
       // Special case when locus has got replaced, This only happend once if a replace locus exists
       // https://sqbu-github.cisco.com/WebExSquared/locus/wiki/Locus-changing-mid-call
 
-      if (data.locus && data.locus.replaces && data.locus.replaces.length > 0) {
+      if (!meeting && data.locus && data.locus.replaces && data.locus.replaces.length > 0) {
```

Comment:
You can use optional chaining here
```suggestion
      if (!meeting && data?.locus?.replaces.length > 0) {
```